openbas-front/src/admin/components/common/form/TextFieldAskAI.tsx

```diff
@@ -398,13 +397,19 @@ const TextFieldAskAI: FunctionComponent<TextFieldAskAiProps> = ({
               <InputLabel id="tone">{t('Tone')}</InputLabel>
               <Select
                 labelId="tone"
-                value={tone}
-                onChange={(event) => setTone(event.target.value as unknown as 'tactical' | 'operational' | 'strategic')}
+                value={messageTone}
+                onChange={(event) => setMessageTone(event.target.value as unknown as 'informal' | 'formal' | 'assertive' | 'sarcastic' | 'authoritative' | 'bitter' | 'critical' | 'arrogant' | 'aggressive')}
                 fullWidth={true}
               >
-                <MenuItem value="tactical">{t('Tactical')}</MenuItem>
-                <MenuItem value="operational">{t('Operational')}</MenuItem>
-                <MenuItem value="strategic">{t('Strategic')}</MenuItem>
+                <MenuItem value="informal">{t('Informal')}</MenuItem>
+                <MenuItem value="formal">{t('Formal')}</MenuItem>
+                <MenuItem value="assertive">{t('Assertive')}</MenuItem>
+                <MenuItem value="sarcastic">{t('Sarcastic')}</MenuItem>
+                <MenuItem value="authoritative">{t('Authoritative')}</MenuItem>
+                <MenuItem value="bitter">{t('Bitter')}</MenuItem>
+                <MenuItem value="critical">{t('Critical')}</MenuItem>
+                <MenuItem value="arrogant">{t('Arrogant')}</MenuItem>
+                <MenuItem value="aggressive">{t('Aggressive')}</MenuItem>
```

Comment:
Nitpick : Maybe we should set them in alphabetic order ?

Reply:
I'll ask the product team